packages/demo-wallet-client/src/ducks/sep31Send.ts

```diff
@@ -324,15 +324,20 @@ export const submitSep31SendTransactionAction = createAsyncThunk<
       });
 
       // Poll transaction until ready
-      await pollTransactionUntilReady({
+      const getSep31Tx = await pollTransactionUntilReady({
```

Comment:
TypeScript is not happy in my VS Code. Can you please try to set a type on `getSep31Tx`? Also, current name sounds like it is a function, maybe we can name it `sep31Tx`?

![image](https://user-images.githubusercontent.com/7346473/163627015-717667b6-1f9c-411b-8346-e4f5b3377922.png)

Reply:
https://github.com/stellar/stellar-demo-wallet/pull/275/commits/5bf0e1caac9386e27bf1fa028723fd0aa14e52f1